src/Command/JsonErrorFormatterDeserializer.php

```diff
@@ -0,0 +1,47 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Command;
+
+use Nette\Utils\Json;
+use Symfony\Component\Console\Command\Command;
+
+class JsonErrorFormatterDeserializer extends Command
```

Comment:
This class doesn't have to extend Command.

Reply:
Removed